lebedigital/openbis/interbis.py

```diff
@@ -57,11 +64,8 @@ def connect_to_datastore(self, username: str = None, password: str = None):
             else:
                 os.environ['OPENBIS_PASSWORD'] = getpass("Give Password: ")
 
-            try:
-                self.login(os.environ['OPENBIS_USERNAME'], os.environ['OPENBIS_PASSWORD'])
-            except ValueError:
```

Comment:
Why do you changed that? Is there already an error if the credentials are wrong in pybis?